webpack/ForemanTasks/Components/TasksTable/TasksTableSchema.js

```diff
@@ -36,24 +36,24 @@ const createTasksTableSchema = (setSort, by, order) => {
   return [
     column(
       'action',
-      __('Action'),
+      'Action',
```

Comment:
What is the reason behind removing the `__`?

Reply:
The `column` translates it and I noticed it here :)
https://github.com/theforeman/foreman/blob/7f113cb4b781283ea7b50ee78d980414dd3bc741/webpack/assets/javascripts/react_app/components/common/table/schemaHelpers/column.js#L28

Reply:
Actually it's bad, the `column` should not do that because we have a script that searches for the `__` in order to extract all the strings needs to be translated and upload them to the translation service.

This way, when you have a variable instead of the actual string, the extraction script won't be able the extract the strings.

IMO, bring back the translations and send a PR to foreman that removes the translation from the column.

Reply:
Added back, will open a PR in foreman